docs/src/staking/README.md

```diff
@@ -35,11 +70,12 @@ line tools, please stay tuned for more information!
 #### Create a Stake Account
 A stake account is a different type of account from a wallet address
 that is used to simply send and receive SOL tokens to other addresses. If you
-have received SOL in a wallet address you control, you can move some of
-these tokens into a new stake account, which will have a different address
-than the wallet you used to create it.  Depending on which wallet you are using
-the steps to create a stake account may vary slightly.  Not all wallets support
-staking operations, see [Supported Wallets](#supported-wallets).
+have received SOL in a wallet address you control, you can use some of
+these tokens to create and fund a new stake account, which will have a different
+address than the wallet you used to create it.
+Depending on which wallet you are using the steps to create a stake account
+may vary slightly.  Not all wallets support stake accounts, see
+[Supported Wallets](#supported-wallets).
```

Comment:
Off topic: We might do well to add a "Wallet vs. Features" grid somewhere in the docs

Reply:
Yeah, good idea.  That would live in the Wallet Guide area.  Certainly worth adding when additional wallets start coming online.